tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/mqtt5/MQTT5Test.java

```diff
@@ -924,4 +924,18 @@ public void testSessionCount() throws Exception {
       producer.connect();
       assertEquals(2, server.getActiveMQServerControl().getSessionCount());
    }
+
+   @Test
+   @Timeout(DEFAULT_TIMEOUT_SEC)
+   public void testSubscriptionQueueRoutingType() throws Exception {
```

Comment:
Should there also be a test that checks expectations for an MQTT subscription that will be using MULTICAST on an address that is already created as ANYCAST ?

Reply:
I believe that already exists at `org.apache.activemq.artemis.tests.integration.mqtt.MQTTTest#testAnycastAddressWorksWithMQTT`.